tests/Functional/Net/SFTPUserStoryTest.php

```diff
@@ -543,14 +543,24 @@ public function testRmDirScratch($sftp)
 
     /**
      * @depends testRmDirScratch
+     * @group github706
```

Comment:
This test does not seem to be a regression test for #706 as it does not fail when the changes above are undone.